ratelearn/ratelearn.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+import pandas as pd
+import torch
+import os
+import argparse
+from torch.utils.data import TensorDataset
+import matplotlib.pyplot as plt
+import logging
+
+from ratelearn import RateMatrix, train_quantization
+
+
+def parse_args():
+    parser = argparse.ArgumentParser("Data2TabShop")
+    parser.add_argument("--frequency_matrices", type=str)
+    parser.add_argument("--stationnary_distribution", type=str, default=None)
+    parser.add_argument("--mask", type=str, default=None)
+    parser.add_argument("--output_dir", type=str)
+    parser.add_argument("--lr", type=float, default=1e-1)
+    parser.add_argument("--do_adam", type=bool, default=True)
+    return parser.parse_args()
+
+
+def get_branch_to_mat(file_path, sep="\s"):
```

Comment:
This method seems to be duplicated from `ratelearn/ratelearner.py`. Should we refactor `ratelearn.py` to use `ratelearn/ratelearner.py` to avoid duplication? Or, remove the `ratelearn.py` script altogether?

Reply:
I agree, let's remove it